Makefile

```diff
@@ -42,55 +42,121 @@ compile:
 	@cd ../java
 	@make
 	@cd ../eo/
-	@eoc dataize app --verbose
 run:
 	@clear
-	@java -cp targets/java/ GraphGenerator 0 10 > tests/edges/star3.graph
-	@java -cp targets/java/ GraphGenerator 1 10 > tests/list/dijkstra2.graph
 	@cd tests/edges
 	@echo "Now we are going to run Kruskal's algorithm \n";
-	@for FILE in *; do \
+	@for FILE in *.graph; do \
 	    var=$$(cat $$FILE)
 		echo "Test ($$FILE) is running"; \
 		$(J); printjava $$(java -cp $(TARGPATH)/java Kruskal $$var); \
 		$(C); printcpp $$($(TARGPATH)/cpp/kruskal $$var); \
+		
+	@cd ../../src/eo/kruskal
+		# $(E); printeo @$$(eoc --easy dataize kruskalApp $$var)
+	@cd ../../../tests/edges/
 		echo "\n";
 	done
 
 	@echo "Now we are going to run Prim's algorithm \n";
-	@for FILE in *; do \
+	@for FILE in *.graph; do \
 	    var=$$(cat $$FILE)
 		echo "Test ($$FILE) is running"; \
 		$(J); printjava $$(java -cp $(TARGPATH)/java prim/PrimMST $$var); \
 		$(C); printcpp $$($(TARGPATH)/cpp/prim $$var); \
 
-	@cd ../../src/eo
-		$(E); printeo $$(eoc --alone dataize app $$var)
-	@cd ../../tests/edges/
+	@cd ../../src/eo/prim
+		# $(E); printeo @$$(eoc --easy dataize primApp $$var)

```

Comment:
@MaxFunProger it's not a good practice to leave commented-out code in the repository. Simply delete it.